playbooks/06-deploy-edpm.yml

```diff
@@ -73,13 +73,13 @@
             name: edpm_deploy
 
 - name: Clear edpm hosts facts to force refreshing in HCI deployments
-  hosts: edpm
+  hosts: computes
```

Comment:
IMHO, we should parameterize as there exists a request to have Ceph deployed on separate nodes.

```
hosts: "{{ cifmw_ceph_target | default('computes')  }}"
```